### ISSUE TITLE
[MC: 1.17.1] [BE: 0.10.5] BetterEnd skybox rendering over end islands at a certain distance

### ISSUE THREAD

**Issue by Merith-TK** [paulevsGitch/BetterEnd#269]
![image](https://user-images.githubusercontent.com/10422110/127777841-ce5aca6c-82ce-4790-a348-d10ac7c45ea9.png)
![image](https://user-images.githubusercontent.com/10422110/127777850-9a34d87c-62e6-4540-bff1-2123f714ff5a.png)
![image](https://user-images.githubusercontent.com/10422110/127777970-b814f750-84fd-4c65-9a6d-6125c709b181.png)

```
Mods:
  [✔️] 1.17-flesh2leather-1.0.2
  [✔️] Adorn-2.1.3+1.17.1-fabric
  [✔️] antiqueatlas-6.0.0-fabric-mc1.17.1
  [✔️] architectury-2.2.21-fabric
  [✔️] atbyw-1.17-1.5.3
  [✔️] backslot-1.2.0
  [✔️] bclib-0.2.4
  [✔️] bedspreads-fabric-1.4.0-1.17
  [✔️] better-end-0.10.5-pre
  [✔️] better-nether-5.1.3
  [✔️] blockus-2.2.7+1.17.1
  [✔️] clickthrough-1.17.1-fabric0.36.1-0.4
  [✔️] Couplings-1.5.1+1.17
  [✔️] Croptopia-1.17-FABRIC-1.6.0
  [✔️] extraorigins-1.17-6
  [✔️] fabric-api-0.37.1+1.17
  [✔️] fabric-furnaces-2.0.1-1.17
  [✔️] fabric-language-kotlin-1.6.2+kotlin.1.5.20
  [✔️] FabricShieldLib-1.3.5-21w05b
  [✔️] figura-0.0.6-1.17.x
  [✔️] FirstPersonMod-2.0.2
  [✔️] gate-of-babylon-1.5.0-1.17
  [✔️] grind-enchantments-1.2.1+1.17
  [✔️] here-be-no-dragons-1.0.0
  [✔️] immersive-portals-0.17-mc1.17.1-fabric
  [✔️] improved-stations-2.0.7
  [✔️] inmis-2.3.1-1.17
  [✔️] InventorySorter-1.7.9-1.17
  [✔️] krypton-0.1.4
  [✔️] medievalweapons-1.1.6
  [✔️] moborigins-1.6.0
  [✔️] modmenu-2.0.4
  [✔️] MoEnchantments-1.17.4
  [✔️] mostructures-1.2.1-1.17
  [✔️] noplayerlabels-2.0.2
  [✔️] notenoughanimations-1.3.0
  [✔️] notenoughcrashes-3.4.0+1.17-fabric
  [✔️] Origins-1.17-1.0.3
  [✔️] Pehkui-2.2.1+1.14.4-1.17
  [✔️] TRansliterationLib-1.1.0
  [✔️] trinketofundying-fabric-2.0.0-1.17
  [✔️] trinkets-3.0.2
```

No errors in logs that are related to better end

**Comment by Merith-TK** [comment]
This would be a *really cool effect* if there was a fog, but this instance is is fresh with only three settings changed

MC GUI Scale = 2
MC Music Vol = 0
MC Autojump = False

**Comment by Merith-TK** [comment]
Changing Render distance does not appear to affect it at all

RD = 8
![image](https://user-images.githubusercontent.com/10422110/127778013-d1e10727-b50e-401d-a969-0cf4f5356725.png)

RD = 12 (default)
![image](https://user-images.githubusercontent.com/10422110/127778026-69b56336-1f20-432b-9963-006ee6c08481.png)

RD = 16
![image](https://user-images.githubusercontent.com/10422110/127778033-7fee8446-b147-4a88-bcc8-7c30a82d1724.png)

For reference, here is my F3 while taking these screenshots
![image](https://user-images.githubusercontent.com/10422110/127778058-fc4b21d5-8614-4bfb-8a22-afe82f150607.png)

And here is what it looks like if i move forward a bit, (all i did was press `w`)
![image](https://user-images.githubusercontent.com/10422110/127778078-c56a5a98-520d-43bb-a56e-324c0086bdd5.png)

**Comment by Merith-TK** [comment]
![image](https://user-images.githubusercontent.com/10422110/127778262-d2fd7198-88f2-46bf-96a5-b963aa1aba44.png)
Standing ***ONE BLOCK BACK FURTHER***
![image](https://user-images.githubusercontent.com/10422110/127778275-9910d5d4-bf9f-46bb-b95c-3da725eedda2.png)

Each line is set at the corner of a chunk

Effective render distance is 7 chunks and 2 blocks

**Comment by paulevsGitch** [comment]
This issue happens with some mods that are not setting sky rendering GL options to default (which vanilla do), the fix was already implemented, but not released yet, and it will be in the next version (0.11.0)

**Comment by paulevsGitch** [comment]
Fixed in 0.11.0
